### ISSUE TITLE
Cannot read propery 'array' of undefined

### ISSUE THREAD

**Issue by xstable** [ccm-innovation/react-native-scrolling-menu#9]
"react": "16.4.1",
"react-native": "^0.56.0",
"react-native-scrolling-menu": "^0.1.1",

![screenshot_1537779904](https://user-images.githubusercontent.com/1778068/45944254-c3ea5680-bfe9-11e8-8096-e384c1923abb.png)

If I deactivate JS-Remote-Debugging on Emulator the message is: 
![screenshot_1537780084](https://user-images.githubusercontent.com/1778068/45944338-1fb4df80-bfea-11e8-8b92-f2383cfd7d40.png)

This is the way I've implement it: 
```JS

              <ScrollingMenu
                items={['Menu Item 1', 'Menu Item 2', 'Menu Item 3', 'Menu Item 4', 'Menu Item 5']}
                backgroundColor="#ffffff"
                textColor="#cccccc"
                selectedTextColor="#000000"
                itemSpacing={20}
              />
```

**Comment by glasje** [comment]
I have the same error ... did you manage to fix it ???

**Comment by xstable** [comment]
Yes, I use this one: https://github.com/experiment322/react-native-scrolling-menu
